main.go

```diff
@@ -168,6 +173,23 @@ func isBitriseCacheAPIURL(url string) bool {
 	return url == os.Getenv("BITRISE_CACHE_API_URL")
 }
 
+func writeCachePullTimestamp() (err error) {
+	f, err := os.Create(cachePullEndTimePath)
+	if err != nil {
+		return err
+	}
+
+	defer func() {
+		if fErr := f.Close(); fErr != nil {
+			err = fErr
+		}
+	}()
+
+	_, err = f.WriteString(strconv.FormatInt(time.Now().UnixNano()/int64(time.Millisecond), 10))
```

Comment:
What would happen if there were multiple cache-pull steps by mistake in a workflow?

Reply:
Last step wins

Reply:
It should be ok, it should not be a common use case. Can you include a note in the release notes later to advise avoiding that situation?